plugins/uefi-capsule/fu-uefi-capsule-plugin.c

```diff
@@ -595,6 +595,24 @@ fu_uefi_capsule_plugin_sort_volume_score_cb(gconstpointer a, gconstpointer b, gp
 	return 0;
 }
 
+static gboolean
+fu_uefi_capsule_volume_is_valid_for_esp(FuVolume *volume)
+{
+	g_autofree gchar *id_type = fu_volume_get_id_type(volume);
+	g_autofree gchar *name = NULL;
+
+	if (g_strcmp0(id_type, "vfat") == 0)
+		return TRUE;
+	/* some BIOS can handle this, but not all */
+	if (g_strcmp0(id_type, "ntfs") == 0)
+		return TRUE;
```

Comment:
is this valid in the spec? I'd be fine leaving it out completely...

Reply:
Okay, if we leave it out then we should pretty much not even add the extra function. I'll respin later on